From: Fabian Vogt <fabian@ritter-vogt.de>
Subject: Do not build parts depending on Qt WebKit

Qt WebKit is unmaintained and must not be used anymore.

Index: kde-workspace-4.11.22/libs/ksysguard/CMakeLists.txt
===================================================================
--- kde-workspace-4.11.22.orig/libs/ksysguard/CMakeLists.txt
+++ kde-workspace-4.11.22/libs/ksysguard/CMakeLists.txt
@@ -17,8 +17,6 @@ configure_file(config-ksysguard.h.cmake
 
 add_subdirectory( lsofui )
 add_subdirectory( processcore )
-add_subdirectory( processui )
-add_subdirectory( signalplotter )
 add_subdirectory( ksgrd )
 add_subdirectory( tests )
 
Index: kde-workspace-4.11.22/plasma/generic/dataengines/CMakeLists.txt
===================================================================
--- kde-workspace-4.11.22.orig/plasma/generic/dataengines/CMakeLists.txt
+++ kde-workspace-4.11.22/plasma/generic/dataengines/CMakeLists.txt
@@ -36,7 +36,6 @@ endif(Boost_FOUND AND KdepimLibs_FOUND)
 
 if(NOT WIN32)
    add_subdirectory(mouse)
-   add_subdirectory(systemmonitor)
    if(${KDE_PLATFORM_PROFILE} STREQUAL "Desktop")
       add_subdirectory(tasks)
    endif(${KDE_PLATFORM_PROFILE} STREQUAL "Desktop")
Index: kde-workspace-4.11.22/CMakeLists.txt
===================================================================
--- kde-workspace-4.11.22.orig/CMakeLists.txt
+++ kde-workspace-4.11.22/CMakeLists.txt
@@ -231,16 +231,12 @@ if(NOT WIN32)
 
    macro_optional_add_subdirectory( powerdevil )
    macro_optional_add_subdirectory( qguiplatformplugin_kde )
-   if (NOT CMAKE_SYSTEM_NAME MATCHES Darwin)
-      macro_optional_add_subdirectory( ksysguard )
-   endif (NOT CMAKE_SYSTEM_NAME MATCHES Darwin)
 endif(NOT WIN32)
 
 if(${KDE_PLATFORM_PROFILE} STREQUAL "Desktop")
    macro_optional_add_subdirectory(kcontrol)
    macro_optional_add_subdirectory(klipper)
    macro_optional_add_subdirectory(kmenuedit)
-   macro_optional_add_subdirectory(krunner)
    macro_optional_add_subdirectory(solid-actions-kcm)
    macro_optional_add_subdirectory(kstartupconfig)
    macro_optional_add_subdirectory(freespacenotifier)
